source/info-tool/ev3dev-system-info.sh

```diff
@@ -32,6 +36,14 @@ print_val() {
   printf "%-20s%s\n" "$1: " "$2"
 }
 
+print_fence_if_markdown() {
+  if [ "$FORMAT_MARKDOWN" = true ]; then
```

Comment:
That isn't a technical limitation, just a shell portability thing -- either way I am happy to change it.